assets/js/blocks/product-categories/get-categories.js

```diff
@@ -2,7 +2,11 @@
  * Internal dependencies
  */
 import { buildTermsTree } from './hierarchy';
-import { PRODUCT_CATEGORIES } from '../../constants';
+
+/**
+ * External dependencies
```

Comment:
We usually list `External dependencies` before `Internal`.

Reply:
See 39ac228